[do not merge] embARC Tool Integration

shea12 opened the pull request.

dfische3 reviewed (approved):
Looks great @shea12!  The only minor question I had was using dpxFileInfo as a class variable as opposed to being passed into individual methods.  But that shouldn't affect function.  Overall it logically makes sense and is well organizes.

shea12 commented:
@dfische3 - good suggestion, I removed the dpxFileInfo class var and passed it through to methods as needed. I did a bit of refactoring and fixed the xml issues we talked about this morning. Ended up implementing the dpx metadata to xml creation on the embARC side. Could you check this out again to make sure it all looks ok?
